lib_xua/api/xua_conf_default.h

```diff
@@ -1729,3 +1728,22 @@ enum USBEndpointNumber_Out
 #define ENUMERATE_CONTROL_INTF_AS_WINUSB    1
 #endif
 
+/**
+ * @brief Macro specifying if mclk is required on the audio tile
+ */
+#define _NEED_MCLK_FOR_I2S \
+  (((I2S_CHANS_DAC != 0) || (I2S_CHANS_ADC != 0)) && (!CODEC_MASTER))
+
+#define _NEED_MCLK_FOR_DIG_TX \
+  (XUA_ADAT_TX_EN || XUA_SPDIF_TX_EN)
+
+#define _NEED_MCLK_FOR_DIG_RX \
+  ((XUA_USE_SW_PLL && XUA_ADAT_RX_EN) || (XUA_USE_SW_PLL && XUA_SPDIF_RX_EN))
+
+#define _NEED_MCLK_FOR_USB \
+  ((AUDIO_IO_TILE == XUD_TILE) && XUA_USB_EN)
+
+/* Final condition */
+#define AUDIO_TILE_MCLK_REQUIRED (_NEED_MCLK_FOR_I2S || _NEED_MCLK_FOR_DIG_TX || _NEED_MCLK_FOR_DIG_RX || _NEED_MCLK_FOR_USB)
+
+#endif /* _XUA_CONF_DEFAULT_H_ */
```

Comment:
This is a nice scheme, can we make if complete by adding info about when the separate mclk for usb is required in the same place?